.env.example

```diff
@@ -39,6 +39,8 @@ FILE_API_KEY_DONCASTER=👻
 FILE_API_KEY_GLOUCESTER=👻
 FILE_API_KEY_TEWKESBURY=👻
 
+# Used to circumvent API rate limiting for development purposes (e.g. load testing)
+SKIP_RATE_LIMIT_SECRET=👻
```

Comment:
I've already generated a 40char alphanumeric secret and pushed it up to S3, so it's ready to pull down